pkg/apiserver/webhooks/datavolume-validate_test.go

```diff
@@ -313,9 +313,109 @@ var _ = Describe("Validating Webhook", func() {
 			resp := validateAdmissionReview(ar)
 			Expect(resp.Allowed).To(Equal(true))
 		})
+
+		It("should accept a spec change on multi-stage import fields", func() {
```

Comment:
Okay, I put these two tests into a DescribeTable. I made test 1 expect to accept the change, since the validator already accepts identical specs otherwise. If the old and new multi-stage fields are the same, it won't try to transfer them again.

Reply:
I also added more unit tests for datavolume-controller, and put some of the VMware calls into an interface so they get tested from new vddk-datasource unit tests. Let me know if you would like anything else in particular to have more tests.